app/models/casa_case_contact_type.rb

```diff
@@ -12,8 +12,8 @@ class CasaCaseContactType < ApplicationRecord
 #  id              :bigint           not null, primary key
 #  created_at      :datetime         not null
 #  updated_at      :datetime         not null
-#  casa_case_id    :integer          not null
-#  contact_type_id :integer          not null
+#  casa_case_id    :bigint           not null
```

Comment:
this change is not intentional and will be reverted by the next person who runs migrations
it is a weird part of our project that this happens...

Reply:
I was about to ask why it works that way. Maybe we might open an issue to investigate it and try to fix it.

Reply:
although it's a mistake from me. I shouldn't have committed those changes. My bad, sorry